bwproject.py

```diff
@@ -231,15 +243,11 @@ def bare_request(verb, address_root, address_suffix, access_token="",
         except Exception as e:
             logger.error("Something was wrong getting a response from "
                          "URL %s" % url)
-            raise BrandwatchApiException(str(e))
+            raise exc.BrandwatchApiException(str(e))
         else:
             errors = response_json.get('errors')
             if errors:
-                logger.error("There was an error with this "
-                             "request: \n{}\n{}\n{}".format(response.url, data,
-                                                            errors))
-                raise BrandwatchApiException(errors)
-
+                _raise_bw_exception(response, data, errors[0])
```

Comment:
We better iterate on the errors since it’s a list.

Reply:
I can only raise one exception :D but I can change exception format if there are several errors